src/erlang_ls_protocol_impl.erl

```diff
@@ -27,11 +27,11 @@
   {response, map()} | {} | {notification, binary(), map()}.
 initialize(Params) ->
   #{ <<"rootUri">> := RootUri
-   , <<"initializationOptions">> := InitOptions
      %% TODO: Store client capabilities, use them in completion_provider
      %%       to verify when Context is present
    , <<"capabilities">> := _ClientCapabilities
    } = Params,
+  InitOptions = maps:get(<<"initializationOptions">>, Params, []),
```

Comment:
I think the default value should be an empty map `#{}` instead of the empty list. The reason why it's not failing is because there is a catch-all clause in `erlang_ls_config:config_path/1`, but from the spec the function expects a `map()`.